build.psm1

```diff
@@ -835,7 +836,14 @@ function Test-PSPesterResults
     if ([int]$x.'test-results'.failures -gt 0)
     {
         logerror "TEST FAILURES"
-        foreach ( $testfail in $x.SelectNodes('.//test-case[@result = "Failure"]'))
+        # switch between methods, SelectNode is not available on dotnet core
+        if ( "System.Xml.XmlDocumentXPathExtensions" -as "Type" ) {
```

Comment:
changed

---
In reply to: [94879724](https://github.com/PowerShell/PowerShell/pull/2958#discussion_r94879724) [](ancestors = 94879724)